security/pam-reattach/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+name                pam-reattach
+github.setup        fabianishere pam_reattach 1.2 v
```

Comment:
please add `github.tarball_from archive` and update the checksums accordingly (i.e., `sudo port clean --all pam-reattach && sudo port -dv checksum pam-reattach`)